mama/c_cpp/src/cpp/MamaDQPublisherManager.cpp

```diff
@@ -190,8 +190,9 @@ struct MamaDQPublisherManagerImpl
     
     MamaDQPublisher* createPublisher (const char *symbol, void * cache);
     void destroyPublisher (const char *symbol);
-    
-      
+
+    MamaDQPublisher* getPublisher (const char* symbol);
```

Comment:
Why this getter doesn't have consts, like const MAMAQDPublisher * const getPublisher (const char* symbol);?